apps/src/netsim/NetSimRouterNode.js

```diff
@@ -633,6 +654,14 @@ NetSimRouterNode.prototype.routeMessage_ = function (message, myWires) {
     return;
   }
 
+  // Automatic DNS: requests to address zero hit the "automatic DNS" system
+  // and generate responses.
+  // TODO (bbuchanan): Send to a real auto-dns node
+  if (this.dnsMode === DnsMode.AUTOMATIC && toAddress === 0) {
```

Comment:
Here, toAddress is the destination address we extracted from the packet header.  Zero is the address of every router in its local network, though we hide this from students for simplicity.

I'll change this to explicitly call out "Addressed to this device" instead of using a magic zero.  Again, this is a temporary quirk of this first pass at auto-dns; that the router IS acting as the DNS node.